### PR TITLE
 ELEMENTS-545: lock datetime-picker dependency

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: widgets/nuxeo-date-picker.html [hunk 1]
```diff
@@ -175,24 +187,26 @@
       },
 
       _pickerValueChanged: function() {
-        if (this._pickerValue === undefined) {
-          return;
-        }
-        if (this._wasOpened && this._pickerValue && moment(this._pickerValue).isValid()) {
-          this.set('value', moment(this._pickerValue).toJSON());
-          this._wasOpened = false;
+        if (this._pickerValue !== undefined && !this._preventPickerUpdate) {
```
Comment (afjusto) [inline widgets/nuxeo-date-picker.html]: `if (this._pickerValue && !this._preventPickerUpdate) {`

File: widgets/nuxeo-date-picker.html [hunk 2]
```diff
@@ -175,24 +187,26 @@
...
+          var date = moment(this._pickerValue);
+          if (this._wasOpened && this._pickerValue && date.isValid()) {
```
Comment (afjusto) [inline widgets/nuxeo-date-picker.html]: you already validate if `this._pickerValue` exists in the first `if` so you shouldn't need to check here

**Comment by guirenard** [comment]
:+1:

File: widgets/nuxeo-date-picker.html [hunk 3]
```diff
@@ -175,24 +187,26 @@
...
 
       _inputValueChanged: function() {
-        if (this._inputValue === undefined) {
-          return;
-        }
-        if (this._inputValue && moment(this._inputValue).isValid()) {
-          this.set('value', moment(this._inputValue).toJSON());
-        } else {
-          this.set('value', '');
+        if (this._inputValue !== undefined && !this._preventInputUpdate) {
```
Comment (afjusto) [inline widgets/nuxeo-date-picker.html]: `if (this._inputValue && !this._preventInputUpdate) {`

File: widgets/nuxeo-date-picker.html [hunk 4]
```diff
@@ -175,24 +187,26 @@
...
+          var date = moment(this._inputValue);
+          if (this._inputValue && date.isValid()) {
```
Comment (afjusto) [inline widgets/nuxeo-date-picker.html]: you already validate if `this._inputValue` exists in the first if so you shouldn't need to check here

**Comment by afjusto** [comment]
👍

**Pull request merged**